src/components/Core/ColoredChip.vue

```diff
@@ -26,8 +26,13 @@ function getThemeColor(color: string) {
   return current.value.colors[color] ?? color
 }
 
-const chipColor = computed(() => (props.disabled || !enableHashColors.value ? props.defaultColor : getColorFromName(props.value)))
-const rawChipColor = computed(() => (props.disabled || !enableHashColors.value ? getThemeColor(props.defaultColor) : getColorFromName(props.value)))
+// Darken colors for OLED theme
+const vueTorrentStore = useVueTorrentStore()
+let darkenColors = false
+if (vueTorrentStore.theme.mode === 'dark' && vueTorrentStore.theme.dark === 'dark-oled') darkenColors = true
+
+const chipColor = computed(() => (props.disabled || !enableHashColors.value ? props.defaultColor : getColorFromName(props.value, darkenColors)))
+const rawChipColor = computed(() => (props.disabled || !enableHashColors.value ? getThemeColor(props.defaultColor) : getColorFromName(props.value, darkenColors)))
```

Comment:
Convert this to a computed property to conserve reactivity (when changing themes)

```suggestion
const isDarkOledTheme = computed(() => vueTorrentStore.theme.mode === 'dark' && vueTorrentStore.theme.dark === 'dark-oled')
const chipColor = computed(() => (props.disabled || !enableHashColors.value ? props.defaultColor : getColorFromName(props.value, isDarkOledTheme.value)))
const rawChipColor = computed(() => (props.disabled || !enableHashColors.value ? getThemeColor(props.defaultColor) : getColorFromName(props.value, isDarkOledTheme.value)))
```